release_tools/README.adoc

```diff
@@ -21,24 +21,26 @@ authentication tokens for these services.
 * Your Jenkins user - your User ID is probably the same as your GitHub handle, to be sure,
   check your *profile page* in Jenkins.
 * A Jenkins token - generate a token on *Configure page* in Jenkins
+  While generating the token, make sure that your Jenkins user has build permissions in Jenkins.
```

Comment:
The users don't have power to grant permissions to themselves.
I guess the message here is that not everyone can start builds from the job page, and they need permission to do so in order to use the automation scripts.